packages/babel-core/test/api.js

```diff
@@ -423,6 +423,8 @@ describe("api", function () {
         "argone;",
         "five;",
         "six;",
+        "twentyone;",
+        "twentytwo;",
```

Comment:
Before the last commit, this was
```js
twentyone
twentytwo
five
six
```

And it was causing a failure in the CRA e2e test.